dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+import "oaidl.idl";
+import "ocidl.idl";
+
+#include "../PushNotifications-Constants.h"
+
+[object]
+[uuid(25604D55-9B17-426F-9D67-2B11B3A65598)]
+[pointer_default(unique)]
+interface IWpnForegroundSink : IUnknown
+{
+    HRESULT InvokeAll([in] ULONG length, [in, size_is(length)] byte* data, [out] BOOL* foregroundHandled);
+};
+
+[object]
+[uuid(60FC21B2-B396-4D49-94F0-7555869FB93C)]
+[pointer_default(unique)]
+interface INotificationsLongRunningPlatform : IUnknown
+{
+    HRESULT RegisterActivator([in] LPCWSTR processName);
```

Comment:
Can we rename this to RegisterLongRunningActivator or something similar? Same comment for the Unregister function call